roles/keystone/templates/etc/keystone/keystone.conf

```diff
@@ -2,8 +2,11 @@
 debug = {{ keystone.logging.debug }}
 verbose = {{ keystone.logging.verbose }}
 
-public_endpoint = https://{{ endpoints.keystone }}:5001/
-admin_endpoint = https://{{ endpoints.keystone }}:35358/
+public_endpoint = {{ endpoints.keystone.url.internal }}/
```

Comment:
`public_endpoint` should use `endpoints.keystone.url.public` instead of `internal`.